packages/gitbook/src/lib/visitors.ts

```diff
@@ -47,7 +48,10 @@ type ClaimPrimitive =
  */
 export type VisitorPayloadLookup = {
     visitorToken: VisitorTokenLookup;
-    unsignedClaims: Record<string, ClaimPrimitive>;
+    unsignedClaims: {
+        all: Record<string, ClaimPrimitive>;
+        fromVisitorParams: Record<string, ClaimPrimitive>;
+    };
```

Comment:
I feel we should only be returning `visitorClaims` which are all the `unsigned` claims. The distinction should not be needed for the caller. 

And then like I said above it should also return `visitorPublicCookie` (if any) that contains claims from the params etc